ver_cli.py

```diff
@@ -257,17 +259,18 @@ def add_database(self, name, db_type, host, port, db_name, username, password):
     # ----------------------------------------------------------------------
     # Profile Functions
 
-    def profile(self, sources_file_name, output_path):
+    def profile(self, sources_file_name, output_path, store_type: int = 1):
```

Comment:
I think it will be better to have the default value as 3, what do you think?

Reply:
It might be good for QuickStart case, but I don't know about the general case though. I think it doesn't matter much if I just add more explanation in the markdown.

Reply:
I decided to change the default to 3 because it's the default value for the ddprofller.